Packs/IBMResilientSystems/Integrations/IBMResilientSystems/README.md

```diff
@@ -12,7 +12,7 @@
 <ul>
 <li><strong>Name</strong>: a textual name for the integration instance</li>
 <li><strong>Server URL</strong></li>
-<li><strong>Credentials</strong></li>
+<li><strong>Credentials (either username and password or API key ID and API key secret)</strong></li>
```

Comment:
need to add to the readme and the details description how to get the api key and api secret

Reply:
I'll see where can I find these details